src/SprykerEco/Client/Payone/Zed/PayoneStub.php

```diff
@@ -158,4 +160,17 @@ public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): Consumer
     {
         return $this->zedStub->call('/payone/gateway/send-consumer-score-request', $quoteTransfer);
     }
+
+    /**
+     * @uses \SprykerEco\Zed\Payone\Communication\Controller\GatewayController::startKlarnaSessionAction()
+     *
+     * @param \Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer $payoneKlarnaStartSessionRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\PayoneKlarnaStartSessionResponseTransfer
+     */
+    public function startKlarnaSession(
+        PayoneKlarnaStartSessionRequestTransfer $payoneKlarnaStartSessionRequestTransfer
+    ): PayoneKlarnaStartSessionResponseTransfer {
+        return $this->zedStub->call('/payone/gateway/start-klarna-session', $payoneKlarnaStartSessionRequestTransfer);
```

Comment:
* Change to '/payone/gateway/send-klarna-start-session-request'.

Reply:
++